### PR TITLE
Task/ahh 116 - You need a food permit component

### PR DESCRIPTION
**Pull request by tim0thysun**
When food requires a food permit in order to be shared, this page will be presented to the host.

### CODE REVIEW

File: foodalert/static/foodalert/js/base.js [hunk 1]
```diff
@@ -31,6 +35,8 @@ window.vueData.routes = [
             { path: "subscribed", component: UnauthzTemplate },
             { path: "audit", component: UnauthzTemplate },
             { path: "welcome", component: WelcomeTemplate },
+            { path: "close", component: CloseTemplate },
+            { path: "need-permit", component: NeedPermit },
```
Comment (abztrakt) [inline foodalert/static/foodalert/js/base.js]: Need to change CloseTemplate to UnauthzTemplate here. Sarthak may just go ahead and fix this.

File: foodalert/static/foodalert/js/base.js [hunk 2]
```diff
@@ -12,6 +14,8 @@ window.vueData.components = {
     "not-found-template": NotFoundTemplate,
     "unauthz-template": UnauthzTemplate,
     "welcome-template": WelcomeTemplate,
+    "close-template": CloseTemplate,
```
Comment (abztrakt) [inline foodalert/static/foodalert/js/base.js]: Not sure if this is needed as it shows up elsewhere.

File: foodalert/static/foodalert/js/base.js (hunk 1 above)
Comment (SarthakSingh31) [reply foodalert/static/foodalert/js/base.js]: This is fixed the last commit

File: foodalert/static/foodalert/js/base.js (hunk 2 above)
Comment (SarthakSingh31) [reply foodalert/static/foodalert/js/base.js]: Removed in the last commit

**Pull request merged**